homework/index.js

```diff
@@ -0,0 +1,32 @@
+const status = (response) => {
+  if (response.status >= 200 && response.status < 300) {
+    return Promise.resolve(response);
+  }
+  return Promise.reject(new Error(response.statusText));
+};
+const json = response => response.json();
+window.onload = () => fetch('https://api.github.com/orgs/HackYourFuture/repos?per_page=100')
+  .then(status)
+  .then(json)
+  .then((data) => {
+    const names = data.map(p => p.name);
```

Comment:
Nit-pick: Actually, doing this intermediary step of mapping the repos into just their name isn't adding much. You could just loop over the data directly, and access its name to fill the inner html & value of the option elements.